src/components/table/table-cell.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './table.css';
+
+export default function TableCell({ children, className, header }) {
+  const Element = header ? 'th' : 'td';
+
+  return (
+    <Element className={className}>
+      {children}
+    </Element>
```

Comment:
Note that I made the decision to re-use `TableCell` for both `td` and `th`.

There's certainly an argument to made **_against_** that.

Reply:
Yeah, you could keep `TableCell` and allow it to take TH || TD and then provide those as decorators e.g. you'd also have a `TableHeader` and a `TableColumn`